tests/sanity/test/install/test-installed-files.js

```diff
@@ -31,6 +37,27 @@ describe('verify installed files', function() {
     await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/components/jobs-api/bin/jobs-api-server-*.jar`);
   });
 
+  it('fingerprint directory should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -d ${process.env.ZOWE_ROOT_DIR}/fingerprint`);
+  });
+
+  it('fingerprint RefRuntimeHash-*.txt should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/fingerprint/RefRuntimeHash-*.txt`);
+  });
+
+  // # verify the checksums of ROOT_DIR, to self-check zowe-verify-authenticity.sh
+  // $binDir/zowe-verify-authenticity.sh # No parameters! 
+  // You need to 'source' the profile to get JAVA_HOME
+
+  it('fingerprint should match', async function() {
+    await sshHelper.executeCommandWithNoError(`. ~/.profile && ${process.env.ZOWE_ROOT_DIR}/bin/zowe-verify-authenticity.sh | tee ~/zowe-verify-authenticity.txt`);
+  });
+
+  expect(file('~/zowe-verify-authenticity.txt')).to.contain('Number of files different =  0');
+  expect(file('~/zowe-verify-authenticity.txt')).to.contain('Number of files extra     =  0');
+  expect(file('~/zowe-verify-authenticity.txt')).to.contain('Number of files missing   =  0');
+  expect(file('~/zowe-verify-authenticity.txt')).to.contain('Verification PASSED');
```

Comment:
I assume `file('~/zowe-verify-authenticity.txt')` will read the content of the file, so it's better to put into a variable other than read it 4 times from file system.

But reading from stdout output is better.

Reply:
Now uses stdout